portfolio/src/main/java/data/Comment.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Comment {
```

Comment:
Consider using AutoValue for entity class. see https://github.com/google/auto/blob/master/value/userguide/index.md